FitNesseRoot/DbFit/AcceptanceTests/CoreTests/CommonSuite/CommitAndRollback/content.txt

```diff
@@ -1,35 +0,0 @@
-!|Rollback|
```

Comment:
As far as I see this file is removed, but not moved to somewhere else. Should it be that way?